Source/WebCore/loader/cache/CachedResource.cpp

```diff
@@ -905,7 +903,7 @@ bool CachedResource::varyHeaderValuesMatch(const ResourceRequest& request)
 unsigned CachedResource::overheadSize() const
 {
     static const int kAverageClientsHashMapSize = 384;
-    return sizeof(CachedResource) + m_response.memoryUsage() + kAverageClientsHashMapSize + m_resourceRequest.url().string().length() * 2;
+    return sizeof(CachedResource) + response().memoryUsage() + kAverageClientsHashMapSize + m_resourceRequest.url().string().length() * 2;
```

Comment:
Should we only count `response().memoryUsage()` if it's not the static one?

Reply:
Unfortunately we can't, since the MemoryCache uses this result as a key, and it can't change without corrupting the cache index.